web/client/codechecker_client/cmd/store.py

```diff
@@ -143,6 +143,23 @@ def add_arguments_to_parser(parser):
                              "If multiple prefix is given, the longest match "
                              "will be removed.")
 
+    parser.add_argument('--config',
+                        dest='config_file',
+                        required=False,
+                        help="Allow the configuration from an explicit JSON "
+                             "based configuration file. The value of the "
```

Comment:
I would probably rephrase this to something like this: the values configured in the config file will overwrite the values set in the command line.